test/kubemark/stop-kubemark.sh

```diff
@@ -37,3 +37,5 @@ gcloud compute disks delete \
       --quiet \
       --zone "${ZONE}" \
       "${MASTER_NAME}"-pd
+
+rm -rf "kubeconfig.loc" &> /dev/null || true
```

Comment:
We don't put newlines at the end of the file. With @wojtek-t we understood the meaning of this red arrow - it's the problem with sublime, which does something strange at the end of the line. Adding a new line and deleting it again in vim fixes the issue.